### PR TITLE
Goodbye Heroku, hello Railway

### PR DESCRIPTION
**Pull request by ItsGoldeneyes**
Now that Heroku's free tier is gone, Railway has become my go-to recommendation to new users.

Removed Heroku - Free tier is no more, don't think it warrants a recommendation anymore.

Replaced with Railway - Similar system to Heroku, free tier is great, developer tier is free for majority of snakes, just needs credit card to prevent fraud. Very helpful support channels in Discord, easy to get started for people who are new to hosting. Would put it above DigitalOcean if they weren't partnered with BattleSnake

### CODE REVIEW

**Comment by bvanvugt** [comment]
Let's keep Heroku listed, but updated to reflect the new pricing. Otherwise looks good!

Also willing to merge this if you just want to add the Railway info and leave Heroku untouched 👍

**Comment by ItsGoldeneyes** [comment]
Let's leave Heroku out, there are some better free options on the page already. Going through the repo now and removing mentions of Heroku

**Comment by ItsGoldeneyes** [comment]
Totally misread your message, thought you said to leave Heroku out, not untouched. I'll fix it

**Comment by ItsGoldeneyes** [comment]
Fixed it, my mistake

File: docs/guides/tips/engine-regions.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ Here are some tips for common hosting setups:
 * e.g. for AWS us-east-1, use "GCP US-EAST4".
 * for DigitalOcean TOR1, use "DigitalOcean TOR (Toronto)"
 
+#### Running on Railway
+
+* Railway is hosted entirely on GCP US-WEST. use the "AWS US-WEST-2" region.
```
Comment (bvanvugt) [inline docs/guides/tips/engine-regions.md:58]: Small typo, 'use' vs. 'Use'

**Review by bvanvugt — CHANGES_REQUESTED** [review]

**Review by bvanvugt — APPROVED** [review]

**Pull request merged**